addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1193,7 +1209,7 @@ export class PosStore extends Reactive {
         // check back-end method `get_product_info_pos` to see what it returns
         // We do this so it's easier to override the value returned and use it in the component template later
         const productInfo = await this.data.call("product.product", "get_product_info_pos", [
-            [product.id],
+            [product?.id],
```

Comment:
So if product is undefined you will make a call with undefined as parameter ? That seems like a bad idea no ?

Reply:
Its compute taxes and stuff still relevent infos